datagenerator/src/main/resources/rythm/common_header.rythm

```diff
@@ -1 +1,7 @@
+@args String type
+
 // DO NOT EDIT - THIS FILE IS GENERATED BY restli-csharp
```

Comment:
In common_header, we may also include information about which pdsc file this .cs file is generated from. Something like this:
// Generated from com/linkedin/restli/objc/integration/SimpleRecord.pdsc
But note that the path to the PDSC file should be relative, not absolute path containing your local file system information.